Examples/Python/python/acts/examples/geant4/__init__.py

```diff
@@ -0,0 +1,23 @@
+import multiprocessing
+
+
+# Cannot conveniently catch linker errors, so we launch a suprocess to
+# try importing and see if it works in order to provide a useful error message
+def _import_test():
+    from acts import ActsPythonBindingsGeant4
+
+
+p = multiprocessing.Process(target=_import_test)
+p.start()
+p.join()
+if p.exitcode != 0:
+    raise RuntimeError(
+        "Error encountered importing Geant4. Likely you need to source $G4DIR/bin/geant4.sh."
+    )
```

Comment:
In the previous PR we had to modify those lines because they were causing some crash. Is it fine for G4 ?
Maybe for consistency you should use the same check as in dd4hep.py ?

Reply:
Ah yeah I had missed this spot. I'll add it.

Reply:
Should be there now.